packages/ethereum/cardstack/transaction-indexer.js

```diff
@@ -313,7 +314,7 @@ class TransactionIndexer {
 
     let addressResource;
     try {
-      addressResource = await this.searchers.getFromControllingBranch(Session.INTERNAL_PRIVILEGED, 'ethereum-addresses', address.toLowerCase());
+      addressResource = await this.searchers.get(Session.INTERNAL_PRIVILEGED, 'local-hub', 'ethereum-addresses', address.toLowerCase());
```

Comment:
@robustdj here is an example of what the new searchers.get() looks like. there is no more `getFromControllingBranch()` if you see any of those change them to just `get()`, and provide a parameter called `local-hub` as the 2nd parameter (we don't use this yet, but this is how we'll be able to target foreign hubs when getting stuff form the searcher).